From: Heiko Stuebner <heiko@sntech.de>
Date: Sun, 24 Jan 2016 00:32:55 +0100
Subject: [PATCH] Add _nl_socket_set_local_port_no_release stub

libnl commit eaa75b7c7d3e ("socket: fix assertion in nl_connect() when all
ports are already in use") removed _nl_socket_generate_local_port_no_release
in favor of a more flexible variant.

Readd a stub function to make our symbols happy
---

--- a/lib/socket.c
+++ b/lib/socket.c
@@ -357,6 +357,11 @@
 }
 /** \endcond */
 
+uint32_t _nl_socket_generate_local_port_no_release(struct nl_sock *sk)
+{
+	return _nl_socket_set_local_port_no_release(sk, 1);
+}
+
 /**
  * @name Source Idenficiation
  * @{
